src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.Security.tests.ps1

```diff
@@ -57,9 +57,13 @@ InModuleScope Arcus.Scripting.Security {
                     -LockLevel CanNotDelete `
                     -Force
 
+                Start-Sleep -Seconds 10
```

Comment:
Why was these delays added? I think we have another function somewhere that polls for stuff, instead of a fixed wait time, if some resource is slow.

Reply:
Don't know by heart, something with `Poll`, I believe. Shame that we had to add these, maybe we can investigate later on how to improve it.

Reply:
Indeed, lets pick this up later on👍🏻

Reply:
Can you create an issue for this an link it here? Thx a lot!